src/odemis/driver/hamamatsurx.py

```diff
@@ -672,6 +677,34 @@ def _setPLLMode(self, value: bool):
 
         return value
 
+    def GetShutter(self) -> bool:
+        """
+        Get the current value from the trigger delay HW (RemoteEx: delay A).
+        :return: (bool) current state of the shutter
+        """
+        shutter_raw = self.parent.DevParamGet(self.location, "Shutter")  # returns a list
+        if shutter_raw[0] == "Open":
+            shutter = True
+            self.state._set_value(model.ST_RUNNING, force_write=True)
+        elif shutter_raw[0] == "Closed":
+            shutter = False
+        else:
+            logging.warning("Unexpected streak mode %s", shutter_raw)
+            shutter = False  # safer!
+
+        return shutter
+
+    def _setShutter(self, value: bool) -> bool:
+        """
+        Updates the shutter state VA.
+        :param value: value to be set
+        :return: (bool) shutter state
+        """
+        self.parent.DevParamSet(self.location, "Shutter", "Open" if value else "Closed")
+        logging.debug("Reporting mechanical shutter state: %s.", value)
```

Comment:
I don't think this sentence is matching. I'd just write "Setting shutter to ...".